packages/sit-onyx/src/composables/checkAll.ts

```diff
@@ -0,0 +1,47 @@
+import type { OnyxCheckboxProps } from "src/components/OnyxCheckbox/types";
+import { computed, type Ref } from "vue";
+
+export type OptionValue = string | number | boolean;
```

Comment:
We can reuse the `SelectionOptionValue` type

Reply:
weird, the first time I did this I got TS errors all over Listbox. now it works. alright ^^